pr-data.csv

```diff
@@ -2729,47 +2729,47 @@ https://github.com/apache/servicecomb-java-chassis,76813bc18a2513d6797794721d0bc
-https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.emptySwaggerDefinitionMediaType,ID,,,
-https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_apikey,ID,,,
-https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_basic,ID,,,
-https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_oauth2,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/731
+https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.ApiProcessorTest.processNotOverWriteValidConsumesAndProduces,ID,Deleted,,https://github.com/apache/servicecomb-java-chassis/commit/1e5cc30515bb9e047376da2ee3410adda9f1a0d2
+https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.emptySwaggerDefinitionMediaType,ID,Deleted,,https://github.com/apache/servicecomb-java-chassis/commit/1e5cc30515bb9e047376da2ee3410adda9f1a0d2
+https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_apikey,ID,Deleted,,https://github.com/apache/servicecomb-java-chassis/commit/1e5cc30515bb9e047376da2ee3410adda9f1a0d2
+https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_basic,ID,Deleted,,https://github.com/apache/servicecomb-java-chassis/commit/1e5cc30515bb9e047376da2ee3410adda9f1a0d2
+https://github.com/apache/servicecomb-java-chassis,eb2961c6b4339c1c34622241cfb5f730fcd35a13,swagger/swagger-generator,org.apache.servicecomb.swagger.generator.core.processor.annotation.SwaggerDefinitionProcessorTest.should_process_security_oauth2,ID,Deleted,,https://github.com/apache/servicecomb-java-chassis/commit/1e5cc30515bb9e047376da2ee3410adda9f1a0d2
```

Comment:
Can you keep the old issue here, not that I exactly understand it at a glance.

Reply:
I added it again

Reply:
Sorry I was not precise: keep the old issue *in addition to* (not instead of) yours.